EIPS/eip-5503.md

```diff
@@ -0,0 +1,176 @@
+---
+eip: 5503
+title: Refundable Token Standard
+author: <admin@startfund.io>
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
[editorial] don't forget to add `discussions-to` ha.

Reply:
added discussions-to item.